### ISSUE TITLE
Quick suggestion and screen reader

### ISSUE THREAD

**Issue by sine-w** [microsoft/vscode#141270]
Hello,
say for example I have the following text content:
--------
```
class C:
    def historical():
      pass
```
if I now type C.his, my screen reader will not speak there is a quick suggestion.
But actually there is a quick suggestion, because if I press enter, C.his will become C.historical.

I checked that if there is more than one suggesttions, screen reader nvda will correctly anounce quick suggestions.

**Comment by jrieken** [comment]
What respective versions of VS Code, the operating system, and nvda are you using?

**Comment by sine-w** [comment]
VS Code version: 1.63.2
Windows version: Windows 21h1 Home ed.
NVDA version: 2021.3.1

**Comment by isidorn** [comment]
I just tried to reproduce this and I could not. For reference here's my full python file. And the end when I type `c.` NVDA properly announces that.

@sine-w I think the problem is the following, when you type `c.h` -> suggest widget shows and NVDA properly announces that, after you add `is` so you have `c.his` NVDA is not saying suggest widget is still visible and that you find confusing? Am I correct in my assessment? If yes do you have suggestions how we could improve this. Since we do announce when the suggest widget gets shown, we do not announce that it is still visible. Maybe we should announce when we hide it? Though that might be too noisy...



```
class C:
	def historical():
		pass

def main():
	c = C()
	c.his
```

**Comment by sine-w** [comment]
Oh, I try to do the things again,
this time when I type c.his, I wait around a second before I type the next character, so like c.h wait 1 sec, i wait 1 sec and s
So if I do it this way I really hear that nvda has anounced suggestion.
in fact, when I type c.h nvda has already said that there is suggestion
But if I type in my normal speed c.his I will not hear any prompt for suggestion.

But when there is more than one suggestion, even if I type in my normal speed, nvda will still report that there are suggestions.

**Comment by isidorn** [comment]
@sine-w I find it strange that this would be connected with the number of items in the suggest widget.
I just think that when you type fast that you interrupt the NVDA announcement.

**Comment by sine-w** [comment]
Hello,
You are right and also wrong.
Probably I have to explain in a more detailed way.

In fact, no matter how many suggestions there are, if you do not type in a fast speed, you will also be able to hear that nvda speak 'suggest,' on the opposite if you type fast no matter how many suggestions there are, you may not be able to hear the 'suggest' because the keyboard will interrupt the nvda speech of 'suggest.'
So the problem here comes, in my real usage, I always type in a very fast speed, so I will not be able to hear the nvda speech of 'suggest' most of the time.
But why I say that if having only one suggestion is different from that of having more than one suggestions ,considered that in both cases the nvda speech of 'suggest' are also being interrupted:
1, with >1 suggestions
even nvda speech of 'suggest' is interrupted, nvda will say '1 of 3' if there are more than one suggestions. So hearing this '1 of #' I will know that there are quick suggestions.
Also, if I press up or down arrow, I can move between different suggestions.
2. with only 1 suggestion
as nvda speech of 'suggest' is interrupted, and there is only one suggestion, nvda will not say anything , so I don't have any other hint to know that there is quick suggestion if the speech 'suggest' is interrupted.
If I move up or down arrow, the cursor will immediately move to the upper line or lower line.

Another piece of information for your reference, the notepad++ that I am using, has a nvda addon, it will generate an audio alert when there are suggestions, so there is no problem of whether the speech will be interrupted or not.

If there are other screen reader users here, please also leave your comments if you have different idea.

**Comment by sine-w** [comment]
Noted the latest update has included Audio cues for screen reader, https://code.visualstudio.com/updates/v1_64
Is it possible that  audio cues be triggered when quick suggestion pop up?

**Comment by isidorn** [comment]
@sine-w we could, but I think it would bring too much noise as the suggestion pop up very often. I think we need to fix the initial issue and that is that the suggestions popping up do not get read to you

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
